doc/Language/variables.rakudoc

```diff
@@ -1370,6 +1370,31 @@ all available resource files in a distribution by printing or by using other
 ways to inspect its value. Instead, use the API described above to
 access particular files.
 
+The C<%?RESOURCES> variable is only accessible inside of modules.
+If you want to access C<%?RESOURCES> outside of a module, you'll need
+to expose that API yourself.  One way to do that is to create a 
+routine in the C<lib> directory to
+return its value:
```

Comment:
Looks like this could go on the previous line

Reply:
So corrected.